website/content/partials/deprecation/internal-counters-tokens-api.mdx

```diff
@@ -0,0 +1,8 @@
+## Internal counters tokens API ((#internal-counters-tokens-api))
+
+| Announced | Expected end of support | Expected removal |
+| :-------: | :---------------------: | :--------------: |
+| N/A       | MAY 2025                | N/A
```

Comment:
```suggestion
| Announced | Expected end of support | Expected removal |
| :-------: | :---------------------: | :--------------: |
| MAY 2025  | MAY 2025                | MAY 2025
```

"N/A" isn't appropriate. We're basically basically announcing the deprecation, ending support, and removing it all at the same time.